src/rpc/methods/eth.rs

```diff
@@ -1387,4 +1387,42 @@ mod test {
         let block = Block::new(true);
         assert_eq!(block.transactions_root, Hash::default());
     }
+
+    // test the case where `tx_arg.to` is `None`
+    #[test]
+    fn test_eth_tx_args_from_unsigned_eth_message() {
+        use crate::shim::address::Address;
+        use fvm_ipld_encoding::RawBytes;
+
+        let msg = Message {
+            version: 0,
+            from: Address::from_str("f410fxcemznbd52stbbtwdm4ayh3edyyyksnkmjbadva").unwrap(),
+            to: Address::from_str("f010").unwrap(),
+            sequence: 4,
+            value: TokenAmount::default(),
+            method_num: 4,
```

Comment:
How are the parameters to this test generated? Are all of them random-ish or do they convey a particular meaning? For example, what happens if I change the `method_num` to 10, `sequence` to 42 and `version` to 3? Will the binary blob below change? It'd be helpful to add some comments here.

Reply:
It's an input payload that generates a `to` address that is `None`.

The payload was extracted using our tool `api compare` with `--n-tipsets 3000`, and I grabbed the ones that fit the bill.
That's a good point. I will add some comments. We only care about the `to` member being `None`, so an assertion of this property will suffice.